tests/integration/false_negative_test.py

```diff
@@ -58,8 +58,9 @@ def test_FalseNegativeCausesNegativeScore(self):
 
     scorer = Scorer(timestamps, predictions, labels, windows, costMatrix,
       probationaryPeriod=0)
+    (_,score) = scorer.getScore()
```

Comment:
nit: space after the comma. Parens around unpacked variables are unnecessary but that's a more of a personal taste I guess